web/controllers/registration_controller.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Yggdrasil.RegistrationController do
+  use Yggdrasil.Web, :controller
+
+  import Ecto.Changeset, only: [put_change: 3]
+  import Comeonin.Bcrypt, only: [hashpwsalt: 1]
+
+  alias Yggdrasil.User
+
+  # if logged in, go straight to client
+  plug Yggdrasil.Plug.SkipIfAuthenticated
+  plug :scrub_params, "user" when action in [:create]
+
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
```

Comment:
I think you can skip it largely for the new function in the render I noticed. But rendering with the changeset does include errors so if you look at registration's create function all the error in the changeset are included in the render. So it's useful to keep the changeset I think in some contexts.